README.md

```diff
@@ -54,6 +54,39 @@ When adding a new pre-commit check, please run Pre-Commit manually first:
 
     % pre-commit run --all-files
 
+### Pull Request Methodology
+We would like, as much as possible, to maintain a linear git history,
+since it is simple, readable, and easier to bisect in case of bugs.
+We would also like to have pull requests reviewed, and finally merged
+responsibly.
+Therefore, our recommendation to all contributors is to follow the following
+procedure:
+1.  Create a [fork](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/working-with-forks/fork-a-repo)
```

Comment:
Step 6 as I understand it assumes that the remote pf the fork is called "origin" and does not rely on there being a name for the fork. Step 7 as I understand it assumes that "upstream" is indeed 'herd/herdtools7" on Github.

So, maybe this could be something like "Create a fork and clone it. Add a remote for the original repository, call it upstream"?